fe/fe-core/src/main/java/com/starrocks/load/ExportJob.java

```diff
@@ -183,7 +184,7 @@ public void setJob(ExportStmt stmt) throws UserException {
 
         exportPath = stmt.getPath();
```

Comment:
I have a question. Is it sure that the exportPath has one and only one trailing '/'?

Reply:
Yes, this is caused by wrong path splicing.